package.json

```diff
@@ -18,7 +18,7 @@
     "gulp": "gulp",
     "karma": "karma",
     "karma.start": "karma start",
-    "postinstall": "typings install && gulp check.versions && npm prune && gulp webdriver",
+    "postinstall": "typings install github:photonstorm/phaser/typescript/typings.json#f2daaec98c095c9b9e4279a618171b051756b895 -GD && typings install && gulp check.versions && npm prune && gulp webdriver",
```

Comment:
this change is unnecessary and was reverted in [this PR](https://github.com/tiagomapmarques/angular2-seed-phaser/pull/2)